ros-ocp/templates/authorino-deployment.yaml

```diff
@@ -0,0 +1,87 @@
+{{- if eq (include "ros-ocp.jwt.shouldEnable" .) "true" }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "ros-ocp.fullname" . }}-authorino
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "ros-ocp.labels" . | nindent 4 }}
+    app.kubernetes.io/component: authorino
+    app.kubernetes.io/part-of: ros-ocp
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "ros-ocp.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: authorino
+  template:
+    metadata:
+      labels:
+        {{- include "ros-ocp.selectorLabels" . | nindent 8 }}
+        app.kubernetes.io/component: authorino
+        authorino-resource: authorino
+        control-plane: controller-manager
+    spec:
+      serviceAccountName: {{ include "ros-ocp.fullname" . }}-authorino
+      containers:
+      - name: authorino
+        image: registry.redhat.io/rhcl-1/authorino-rhel9@sha256:22d44b5bccacac44ffe96206e69ac91e22557f1c4fbfd46252c6ba9b720d7d6a
```

Comment:
not sure: does it make any sense to have this in `values.yaml`? To ease the upgrade.